serialize/internal.thrift

```diff
@@ -0,0 +1,14 @@
+struct RPC {
+	1: required binary spanContext
```

Comment:
I see: baggage carried here. Probably worth clarifying the binary encoding protocol for the span context here, with a reference to the opentracing binary carrier, and verifying that the wire encoding is stable @yurishkuro.

This presumably carries both baggage and tracing.